Description: Increase stack size for specific tests
 This makes those tests pass.

Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed

--- a/test/mjsunit/big-array-literal.js
+++ b/test/mjsunit/big-array-literal.js
@@ -26,7 +26,8 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 // On MacOS, this test needs a stack size of at least 538 kBytes.
-// Flags: --stack-size=600
+// On Debian amd64 the minimum seems to be 763
+// Flags: --stack-size=800
 
 // Test that we can make large object literals that work.
 // Also test that we can attempt to make even larger object literals without
